### DISCUSSION TITLE
Full report offline

### DISCUSSION THREAD

**Discussion by pedddro** [harlan-zw/unlighthouse#53] [General]
I know each report is stored offline and accessible but is there a way to see the full report offline?

**Comment by matthitmag** [comment]
I too would like to know this.

**Comment by harlan-zw** [comment]
Hey! Thanks for the discussion. Would you mind providing some more details on what the issue is? It's not exactly clear.

As far as I understand it the report _should_ work offline. You can generate the report and open it using sirv `npx sirv-cli .unlighthouse`

**Reply by pedddro** [reply]
thanks for the reply, running `npx sirv-cli` on one of the reports and it stays on "Waiting for routes..." with console errors:

```
:5678/api/scan-meta:1     Failed to load resource: net::ERR_CONNECTION_REFUSED
:5678/api/reports:1     Failed to load resource: net::ERR_CONNECTION_REFUSED
```
Is there additional flags to run?

**Reply by harlan-zw** [reply]
When creating the report you need to pass the `--build-static` flag. Then you can run `npx sirv-cli .unlighthouse/client` (iirc). 

There are docs for the CI static usage here: https://unlighthouse.dev/integrations/ci#build-static-report

The documentation could be improved for this, maybe adding a "Static Reports" to the guides section.

**Reply by pedddro** [reply]
Thank you i got it working afters installing the cli and running `npx sirv-cli .unlighthouse`

The documentation can definitely improve for "Static Reports".

**Comment by Robanna777** [comment]
After I run with the --build-static flag, should there be a client directory? I'm not seeing any difference with or without that flag. I get the Waiting for routes... either way.

**Reply by Robanna777** [reply]
I didn’t have the puppeteer installed correctly. It's working now.

**Comment by pedddro** [comment]
@harlan-zw is the website open source? where can I amend the documentation with a PR?

**Reply by harlan-zw** [reply]
A PR would be amazing, you can find the documentation in the `docs` folder https://github.com/harlan-zw/unlighthouse/tree/main/docs. 

All content is markdown so should be easy enough to navigate, let me know if you have ay questions though,
